lib/Avatar/styles.scss

```diff
@@ -35,6 +35,10 @@ $avatar-pillbox-padding: $typo-size-lead/3 $typo-size-lead/2 !default;
   &.is-faded {
     background: $avatar-faded-background !important;
   }
+
+  &.is-assigned {
```

Comment:
It's ideal not to tie names in a UI library too much with business logic. Lets rename this to something a little more generic/abstract. For example adding a modifier such as `avatar--highlight`.